### PR TITLE
Hex Integers

### PR DESCRIPTION
**Pull request by QianyangPeng**
a very short implementation of HEX. Could pass all conformance tests.

### CODE REVIEW

File: tests/simple/i32-c.wast [hunk 1]
```diff
@@ -0,0 +1,959 @@
...
+(assert_return (invoke "ge_u" (i32.const 0x80000000) (i32.const 0)) (i32.const 1))
+(assert_return (invoke "ge_u" (i32.const 0) (i32.const 0x80000000)) (i32.const 0))
+(assert_return (invoke "ge_u" (i32.const 0x80000000) (i32.const -1)) (i32.const 0))
+(assert_return (invoke "ge_u" (i32.const -1) (i32.const 0x80000000)) (i32.const 1))
+(assert_return (invoke "ge_u" (i32.const 0x80000000) (i32.const 0x7fffffff)) (i32.const 1))
+(assert_return (invoke "ge_u" (i32.const 0x7fffffff) (i32.const 0x80000000)) (i32.const 0))
+
+
+;; (assert_invalid
+;;   (module
```
Comment (hjorthjort) [inline tests/simple/i32-c.wast]: Do these need to be commented out?

**Comment by hjorthjort** [comment]
Hmm, I'm getting a `ClassCastException` when compiling this, but somehow the checks passed. That's weird. Investigating

**Comment by hjorthjort** [comment]
I un-commented parts of the conformance tests we can actually parse (it's only the floats we can't parse yet) and pushed. Hope you don't mind. But that's all I need to be convinced that this is fine.

**Comment by QianyangPeng** [comment]
Approved. BTW do you think it will be easier if we just use -1 in place of `.Int`?

File: data.md [hunk 2]
```diff
@@ -185,6 +185,17 @@ The `#width` function returns the bit-width of a given `IValType`.
     rule #pow (i64) => 18446744073709551616
 ```
 
+```k
```
Comment (ehildenb) [inline data.md]: Please add documentation before this code block.

We should probably never have a code block that doesn't have documentation before it.

**Pull request merged**